docs/user/power/install_upi.md

```diff
@@ -0,0 +1,300 @@
+
+3. DNS requirements
+
+4. Getting Ignition configs for machines
+
+5. Getting OS related assets for machines
+
+6. Booting machines with RHCOS and Ignition configs
+
+7. Watching your installation (bootstrap complete, cluster available)
+
+8. Example Bare-Metal UPI deployment
+
+## Compute
+
+The smallest OpenShift 4.x clusters require the following host:
+
+* 1 bootstrap machine.
+
+* 3 control plane machines.
+
+* atleast 1 worker machine.
+
+NOTE: The cluster requires the bootstrap machine to deploy the OpenShift cluster on to the 3 control plane machines, and you can remove the bootstrap machine.
+
+All the machines must use Red Hat Enterprise Linux CoreOS (RHCOS) as the operating system.
+
+### Minimum resource requirements
+
+| Machine       | Operating System | vCPU | RAM   | Storage |
+|---------------|------------------|------|-------|---------|
+| Bootstrap     | RHCOS            | 2    | 16 GB | 120 GB  |
+| Control Plane | RHCOS            | 2    | 16 GB | 120 GB  |
+| Compute       | RHCOS            | 2    | 16 GB | 120 GB  |
+
+
+
+## Network Topology Requirements
+
+The easiest way to get started is to ensure all Power nodes have internet access to pull images for platform containers and provide telemetry data to Red Hat.
+OpenShift 4.x also supports air-gapped installs
+
+### Load balancers
+
+Before you install OpenShift, you must provision two load balancers.
+
+* A load balancer for the control plane and bootstrap machines that targets port 6443 (Kubernetes APIServer) and 22623([Machine Config server][machine-config-server]). Port 6443 must be accessible to both clients external to the cluster and nodes within the cluster, and port 22623 must be accessible to nodes within the cluster.
+
+  NOTE: Bootstrap machine can be deleted as target after cluster installation is finished.
+
+* A load balancer for the machines that run the [ingress router][openshift-router] pods that balances ports 443 and 80. Both the ports must be accessible to both clients external to the cluster and nodes within the cluster.
+
+    NOTE: A working configuration for the ingress router is required for an OpenShift 4.x cluster.
+
+    NOTE: The default configuration for Cluster Ingress Operator  deploys the ingress router to `worker` nodes in the cluster. The administrator needs to configure the [ingress][openshift-router] after the control plane has been bootstrapped.
+
+### Connectivity between machines
+
+You must configure the network connectivity between machines to allow cluster components to communicate.
+
+* etcd
+
+    As the etcd members are located on the control plane machines. Each control plane machine requires connectivity to [etcd server][etcd-ports], [etcd peer][etcd-ports] and [etcd-metrics][etcd-ports] on every other control plane machine.
+
+* OpenShift SDN
+
+    All the machines require connectivity to certain reserved ports on every other machine to establish in-cluster networking. For more details refer [doc][sdn-ports].
```

Comment:
"For more details refer doc." doesn't sound good.

Reply:
Any suggestions ?

Reply:
"For more details refer to the documentation."

Reply:
Changed it to "For further detail, please refer to the following documentation". Hope this is fine